components/rdf-ntriples.js

```diff
@@ -0,0 +1,86 @@
+// rdf-ntriples.js
+
+var _ = require('underscore');
+
+var promiseOutput = require('../src/promise-output');
+var componentFactory = require('../src/noflo-component-factory');
+
+/**
+ * Converts an RDF JS Interface Graph object into an object with the property
+ * 'tokens' containing an Array of N-Triples tokens. tokens.join('') will produce an N-Triples string.
+ */
+exports.getComponent = componentFactory({
+    description: "Converts an RDF JS Interface Graph object into an Array of N-Triples tokens",
+    icon: 'edit',
+    outPorts: promiseOutput.outPorts,
+    inPorts: {
+        input: {
+            description: "RDF JS Interface Graph object",
+            datatype: 'object',
+            required: true,
+            ondata: promiseOutput(execute)
+        }
+    }
+});
+
+/**
+ * Promise of N-Triples array of tokens for the given graph.
+ * @this a noflo.InPort or facade
+ * @param graph RDF JS Interface Graph object
+ */
+function execute(graph) {
+    var tokens = [];
+    graph.forEach(function(triple) {
+        tokens.push.apply(tokens, valueTokens(triple.subject));
+        tokens.push(' ');
+        tokens.push.apply(tokens, valueTokens(triple.predicate));
+        tokens.push(' ');
+        tokens.push.apply(tokens, valueTokens(triple.object));
+        tokens.push(' .', '\n');
+    });
+    return {tokens: tokens};
+}
+
+/**
+ * Converts the RDF term into a array of tokens
+ * @param object an RDF term implementing RDF JS Interface
```

Comment:
parameter name does not match the actual parameter (term). That's a nit, but more interesting would be a bit more documentation on the term object structure - it looks to me like it's coming from some kind of possibly standard source.  If so, can you point to that?